rdr_service/config/base_config.json

```diff
@@ -11,7 +11,10 @@
     "questionnaireOnFamilyHealth",
     "questionnaireOnHealthcareAccess",
     "questionnaireOnMedicalHistory",
-    "questionnaireOnMedications"
+    "questionnaireOnMedications",
+    "questionnaireOnCopeMay",
+    "questionnaireOnCopeJune",
+    "questionnaireOnCopeJuly"
```

Comment:
Why are these being added to the app config?

Reply:
The questionnaire_response_dao uses count_completed_ppi_modules at [line 75](https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/dao/questionnaire_response_dao.py#L75) to update numCompletedPPIModules. The ticket seems to be asking for it.